#@module_name pathGenerator
geometry_msgs/Pose2D start
geometry_msgs/Pose2D goal
geometry_msgs/PoseArray dynamicObstacle
---
geometry_msgs/PoseArray path
std_msgs/Float32 length